### ISSUE TITLE
Merge L-2-aminoadipate N-acetyltransferase activity into GO:0043869 alpha-aminoadipate acetyltransferase activity

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#18958]
Cleaning up IMG: cross references #18908

Hi @hdrabkin 

Are those 2 terms the same ? 

id: GO:0043741
name: L-2-aminoadipate N-acetyltransferase activity
namespace: molecular_function
def: "Catalysis of the reaction: L-2-aminoadipate + acetyl-CoA = N2-acetyl-L-aminoadipate + CoA." [IMG:01447]
xref: MetaCyc:RXN-5181
is_a: GO:0008080 ! N-acetyltransferase activity

=> no annotations

Merge into GO:0043869 alpha-aminoadipate acetyltransferase activity : Catalysis of the reaction: alpha-aminoadipate + acetyl-CoA = N2-acetyl-alpha-aminoadipate + coenzyme A." [MetaCyc:RXN-5181,

=> 1 ZFIN 
=> Metacyc https://biocyc.org/META/NEW-IMAGE?type=NIL&object=RXN-5181
has the substrate for GO:0043741. 

Thanks, Pascale

**Comment by pgaudet** [comment]
BTW both GO terms have the same MetaCyc xref 
MetaCyc:RXN-5181

**Comment by hdrabkin** [comment]
Yes I see the same metacyc.
I'd merge the second into the first (ie, keep the one with the L-2-aminoadipate substrate.
I can request since the metacyc rxn has all the chebi ids I need, but they will require a PMID also.

**Comment by hdrabkin** [comment]
Use the zfin annotation PMID?

**Comment by pgaudet** [comment]
sure

**Comment by pgaudet** [comment]
Should I do it or do you want to ?

**Comment by hdrabkin** [comment]
You can do the merge and mark it RHEA pending 
I just found the ref: PMID:29053735 to add to the def.

**Comment by hdrabkin** [comment]
Rhea requested for merged term.
[help #181278] in the subject line of all future correspondence about this issue.

**Comment by hdrabkin** [comment]
I would leave it open if we are waiting for RHEA..

**Comment by hdrabkin** [comment]
Hi @pgaudet 
Here is RHEA's response
I had supplied the PMID that zfin used to annotate the less specific term for GO:0043869 alpha-aminoadipate; the only PMID we had.

Bottom line: No evidence for the reaction  GO:0043741
Maybe we want to obsolete it. 

Dear Harold,

From PMID:29053735
"ALDH7A1 encodes alpha-aminoadipic semialdehyde dehydrogenase (EC 1.2.1.31) in
the lysine catabolic pathway and the enzyme deficiency leads to accumulation of
alpha-aminoadipic acid semialdehyde (alpha-AASA) and piperideine 6-carboxylate
(P6C), the latter inactivates pyridoxal-5-phosphate."

In UniProt:
https://www.uniprot.org/uniprot/P49419
P49419 (AL7A1_HUMAN)
Alpha-aminoadipic semialdehyde dehydrogenase
has a sequence highly similar to the 2 Zebrafish TrEMBL entries F1QR17_DANRE
(https://www.uniprot.org/uniprot/F1QR17)
Q803R9_DANRE (https://www.uniprot.org/uniprot/Q803R9)

One of the reaction catalyzed by AL7A1_HUMAN is RHEA:12304
(S)-2-amino-6-oxohexanoate + H2O + NADP(+) = 2 H(+) + L-2-aminoadipate + NADPH
(https://www.rhea-db.org/reaction?id=12304)
Note that CHEBI:58321 (S)-2-amino-6-oxohexanoate == L-2-aminoadipate
semi-aldehyde == alpha-AASA
Whereas RXN-5181 (L-2-aminoadipate + acetyl-CoA = N2-acetyl-L-aminoadipate +
CoA) is an N-acetyl transferase reaction, not mentioned in the article.
If I remember well, I think RXN-5181 was a reaction supposed to protect the
amino group of L-aminoadipate in some L-lysine biosynthesis pathway. It was
discovered later that this amino group was in fact protected by binding to LysW
(the reaction catalyzed by LysX I mentioned in my previous mail).
Note that the MetaCyc reaction
(https://metacyc.ai.sri.com/META/NEW-IMAGE?object=RXN-5181) is neither linked
to an article nor to a known enzyme.

**In conclusion, we can't create the requested reaction as long as we don't have
experimental evidence for it.**

I hope it make sense for you and you will understand our concern.

Cheers, Anne

On Wed Feb 26 20:25:06 2020, harold.drabkin@jax.org wrote:

  Hi Ann
  
  It's a zfin gene that used it
  PMID:29053735
  Characterization of the first knock-out aldh7a1 zebrafish model for pyridoxine-dependent epilepsy using CRISPR-Cas9 technology.
  
  We are merging two terms GO:0043869 into GO:0043741
  Both reference the same metacyc id MetaCyc:RXN-5181, and the metacyc is specific for the L-2-aminoadipate
  (43869 merely says 2-aminoadipate
  
  So I thought I'd get a RHEA to match the metacyc.
  
  I have attached the pdf for PMID;
  
  
  On 2/26/20, 2:14 PM, "Anne Morgat via RT" <rhea-reaction@sib.swiss> wrote:
  
      Hi Harold,
      I have some trouble to find the requested reaction in PMID:29053735
      What are the genes annotated with GO:0043741 term?
      
      I can find some TrEMBL entries (unreviewed) still annotated as
      "L-2-aminoadipate N-acetyltransferase"
      Ex: https://www.uniprot.org/uniprot/A0A1G7D5W5
      But A0A1G7D5W5 share 96.1% seq identity with Q5SH23 (LYSX_THET8)
      "Alpha-aminoadipate--LysW ligase LysX"
      RHEA:41940 (EC 6.3.2.43)
      [2-aminoadipate-carrier protein]-C-terminal-L-glutamate + ATP +
      L-2-aminoadipate = [2-aminoadipate-carrier
      protein]-C-terminal-gamma-(L-2-aminoadip-2-yl)-L-glutamate + ADP + H(+) +
      phosphate
      
      Cheers, Anne
      
      On Wed Feb 26 19:11:22 2020, harold.drabkin@jax.org wrote:
      
        (sent by harold.drabkin@jax.org)
        To be used for
        d: GO:0043741
        name: L-2-aminoadipate N-acetyltransferase activity
        namespace: molecular_function
        def: "Catalysis of the reaction: L-2-aminoadipate + acetyl-CoA = N2-acetyl-L-aminoadipate + CoA."
        Metacyc:RXN-5181
        
        L-2-aminoadipate + acetyl-CoA = N2-acetyl-L-aminoadipate + CoA."
        All of these have CHEBI ids
        
        PMID:29053735
         
    
--
Anne Morgat
SIB | Swiss Institute of Bioinformatics
CMU - 1, rue Michel Servet - 1211 Geneva 4
t: +41 22 379 49 25 - f: +33 4 76 61 54 08
Anne.Morgat@sib.swiss - http://www.sib.swiss

**Comment by pgaudet** [comment]
I think the merge is OK.

**Comment by hdrabkin** [comment]
yes as long as GO:0043869 is the winning term

**Comment by pgaudet** [comment]
Ah ! no. I'll swap them.

**Comment by hdrabkin** [comment]
Because the less general term (43869) is the one with the annotation and reference and the one with the  L-2-aminoadipate N-acetyltransferase activity was the one RHEA could not find sufficient evidence for .

**Comment by pgaudet** [comment]
Sure, but this is not 'visible' once everything is merged ? 
<img width="650" alt="image" src="https://user-images.githubusercontent.com/4782928/75590484-6e6e7080-5a4b-11ea-9e6a-4f96baa6bad6.png">
